src/ints/int10_vesa.cpp

```diff
@@ -148,77 +148,81 @@ Bit8u VESA_GetSVGAModeInformation(Bit16u mode,Bit16u seg,Bit16u off) {
 	switch (mblock->type) {
 	case M_LIN4:
 		pageSize = mblock->sheight * mblock->swidth/2;
-		var_write(&minfo.BytesPerScanLine,mblock->swidth/8);
-		var_write(&minfo.NumberOfPlanes,0x4);
-		var_write(&minfo.BitsPerPixel,4);
-		var_write(&minfo.MemoryModel,3);	//ega planar mode
-		modeAttributes = 0x1b;	// Color, graphics, no linear buffer
+		minfo.BytesPerScanLine = host_to_le16(mblock->swidth / 8);
+		minfo.NumberOfPlanes = 0x4;
+		minfo.BitsPerPixel = 4u;
+		minfo.MemoryModel = 3u; // ega planar mode
+		modeAttributes = 0x1b; // Color, graphics, no linear buffer
 		break;
 	case M_LIN8:
 		pageSize = mblock->sheight * mblock->swidth;
-		var_write(&minfo.BytesPerScanLine,mblock->swidth);
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the host_* functions do).